zarnitsa/DataAugmenterExternally.py

```diff
@@ -26,15 +26,13 @@ def augment_dataframe(
             raise KeyError("Bad type of aug_params variable")
         return df
 
-    def augment_column(
-        self, col: pd.Series, aug_type="normal", **kwargs
-    ) -> pd.Series:
+    def augment_column(self, col: pd.Series, aug_type="normal", **kwargs) -> pd.Series:
         """Augment Serial data. Pandas column"""
+        if len(col) == 0:
```

Comment:
it's up to you, but i think that `len(col)` or `col` is sufficient

Reply:
col is not sufficient